spec/features/view_allocations_spec.rb

```diff
@@ -33,4 +33,10 @@
     expect(page).to have_content("New allocations")
     expect(page).to have_content("Abbella, Ozullirn")
   end
+
+  it 'allows staff without the POM role to view the my casload page', vcr: { cassette_name: :non_pom_my_caseload }  do
+    signin_user('NON_POM_GEN')
+    visit "/poms/485767/my_caseload"
```

Comment:
Staff id isn't necessary.  Will need to remove this from the routes in routes.rb for my_caseload and new_cases.